libcloud/compute/drivers/openstack.py

```diff
@@ -2589,6 +2603,43 @@ def ex_update_image(self, image_id, data):
         )
         return self._to_image(response.object)
 
+    def ex_list_image_members(self, image_id):
+        """
+        List all members of an image. See
+        https://developer.openstack.org/api-ref/image/v2/index.html#sharing
+
+        :param      image_id: ID of the image of which the members should
+        be listed
+        :type       image_id: ``str``
+
+        :rtype: ``list`` of :class:`NodeImageMember`
+        """
+        response = self.image_connection.request(
+            '/v2/images/%s/members' % (image_id,)
+        )
+        image_members = []
+        for image_member in response.object['members']:
+            image_members.append(self._to_image_member(image_member))
+        return image_members
+
+    def ex_list_image_member(self, image_id, member_id):
+        """
+        List a member of an image by id
```

Comment:
Is there a good reason to call this operation list and not get?

Reply:
Oops, has been changed to ex_get_image_member now to conform to the convention